synapse_antispam/mjolnir/antispam.py

```diff
@@ -115,6 +115,11 @@ def user_may_create_room_alias(self, user_id, room_alias):
     def user_may_publish_room(self, user_id, room_id):
         return True  # allowed
 
+    def check_username_for_spam(self, user_id, display_name):
```

Comment:
Can we move this up above the no-op functions? Just so it stays reasonably close to where the actual action is

Reply:
Definitely. I had kept them in the same order as the `SpamChecker` caller from Synapse, but that doesn't really matter. 👍